tgui/packages/common/storage.ts

```diff
@@ -7,98 +7,121 @@
  */
 
 export const IMPL_MEMORY = 0;
-export const IMPL_LOCAL_STORAGE = 1;
+export const IMPL_HUB_STORAGE = 1;
 export const IMPL_INDEXED_DB = 2;
 
+type StorageImplementation =
+  | typeof IMPL_MEMORY
+  | typeof IMPL_HUB_STORAGE
+  | typeof IMPL_INDEXED_DB;
+
 const INDEXED_DB_VERSION = 1;
 const INDEXED_DB_NAME = 'tgui';
 const INDEXED_DB_STORE_NAME = 'storage-v1';
 
-const READ_ONLY = 'readonly';
-const READ_WRITE = 'readwrite';
+const READ_ONLY = 'readonly' as const;
+const READ_WRITE = 'readwrite' as const;
```

Comment:
```suggestion
const READ_ONLY = 'readonly';
const READ_WRITE = 'readwrite';
```
`as const` only works on objects and arrays